lib/locales/en/frontend/alert-messages.json

```
@@ -10,7 +10,9 @@
   "error_withdrawing_exists_pending_ones": "No withdraws can be executed while there are pending ones.",
   "withdrawal_amount_being_processed": "The withdrawal of {{amount}} is being processed",
   "two_fa_code": {
+    "title": "Two-Factory Authentication",
     "required": "It's mandatory to have the Two-Factory Authentication method active",
+    "required_with_instruction_to_activate": "It's mandatory to have the Two-Factory Authentication method active. To continue, you must first enable it in <1>settings page</1>.",
```

We can delete this `title` and `required_with_instruction_to_activate`, right?
```suggestion
    "required": "It's mandatory to have the Two-Factory Authentication method active",
```